new-log-viewer/src/components/ConfigForm.tsx

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+
+import Button from "@mui/joy/Button";
+import FormControl from "@mui/joy/FormControl/FormControl";
+import FormLabel from "@mui/joy/FormLabel/FormLabel";
+import Input from "@mui/joy/Input";
+
+import {
+    CONFIG_KEY,
+    LOCAL_STORAGE_KEY,
+} from "../typings/config";
+import {
+    getConfig,
+    setConfig,
+} from "../utils/config";
+
+
+const formFields = [
+    // FIXME: add helper text for each field.
```

Comment:
Do we plan to address this FIXME in the same PR?

Reply:
I don't really know. Do we need a helper text here? Your previous suggestion has one helper text field, but currently we haven't implement that.

Reply:
nvm. I will add helper text field in the formFields

Reply:
let's discuss offline about what exactly the helper texts should be